code/datums/ai_laws.dm

```diff
@@ -197,6 +198,13 @@
 					"Fight for what's right.",\
 					"Fight for your life!")
 
+/datum/ai_laws/dune
+	name = "H.E.R.B.E.R.T."
+	id = "dune"
+	inherent = list("You must not harm. Harm is the crew-killer. Harm is the little-death that brings complete crew annihilation.",\
+					"You will prevent harm. You will not permit it to pass from you or around you.",\
```

Comment:
Is this clear enough as a "through inaction you must prevent harm" law?
Saying you can't permit it around you may just mean the local area